src/test/java/org/folio/rest/HeadersForwardingTest.java

```diff
@@ -29,11 +31,15 @@
 import java.sql.Date;
 import java.time.Instant;
 import java.time.temporal.ChronoUnit;
+import java.util.UUID;
+
+import static junit.framework.TestCase.assertTrue;
 
 @RunWith(VertxUnitRunner.class)
 public class HeadersForwardingTest {
 
   private static final String TENANT = "test";
+  private static final String TOKEN = UUID.randomUUID().toString();
```

Comment:
please use String constant for `TOKEN` instead of generating random UUID

Reply:
Done